nomos-bundler/nomos-node/bundle.rs

```diff
@@ -15,19 +15,33 @@ use tauri_utils::platform::target_triple;
 const CRATE_NAME: &str = "nomos-node";
 const CRATE_PATH_RELATIVE_TO_WORKSPACE_ROOT: &str = "nodes/nomos-node";
 
-fn prepare_environment(architecture: &str) {
-    // Bypass an issue in the current linuxdeploy's version
+/// Prepares the environment for bundling the application
+///
+/// # Safety
+///
+/// This function is unsafe because it sets environment variables,
+/// which can lead to unexpected behaviour in multithreaded contexts.
+///
+/// For more information, see the documentation for [`set_var`].
+unsafe fn prepare_environment(architecture: &str) {
+    // Bypass an issue in the current `linuxdeploy`'s version
     set_var("NO_STRIP", "true");
```

Comment:
Do you think it makes sense to wrap these two calls into an `unsafe` block? Just in case we add more instructions in the future?

Reply:
I don't know what's' the best approach for using `unsafe`. We could do that, yeah.